addons/website_event_crm_questions/models/event_registration.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, _
+
+
+class EventRegistration(models.Model):
+    _inherit = 'event.registration'
+
+    def _get_lead_description(self):
+        """
+        Add to the description of the lead, the questions and answers
+        linked to the registrations
```

Comment:
`Add the questions and answers linked to the registrations into the description of the lead`